mkdocs.yml

```diff
@@ -3,6 +3,7 @@ site_dir: docs/html
 nav:
     - Home: index.md
     - "Quick Start": quick-start.md
+    - "Migrating from v2 to v3": migrating.md
```

Comment:
Everything related to migration must be added at the end of the navigation.

```yaml
- Migration:
   - "Migration from Version 2 to 3": v3/migration/v2-to-v3.md
```

(There will be a new homepage, which will also include the link to the migration guide.)